### PR TITLE
 :electron:  Making Electron server logs visible in devtools

### PR DESCRIPTION
**Pull request by MikesGlitch**
**Adding some error logs:**

Mac issue: 
image

### CODE REVIEW

File: packages/desktop-electron/index.ts [hunk 1]
```diff
@@ -60,9 +60,21 @@ function createBackgroundProcess() {
   serverProcess = utilityProcess.fork(
     __dirname + '/server.js',
     ['--subprocess', app.getVersion()],
-    isDev ? { execArgv: ['--inspect'] } : undefined,
+    isDev ? { execArgv: ['--inspect'], stdio: 'pipe' } : { stdio: 'pipe' },
```
Comment (MikesGlitch) [inline packages/desktop-electron/index.ts:63]: Piping out from the forked server process - allows us to handle the stdout/stderr

File: packages/desktop-electron/index.ts [hunk 2]
```diff
@@ -60,9 +60,21 @@ function createBackgroundProcess() {
...
   );
 
+  serverProcess.stdout.on('data', (chunk: Buffer) => {
+    // Send the Server console.log messages to the main browser window
+    clientWin?.webContents.executeJavaScript(`
+      console.info('Server Log:', ${JSON.stringify(chunk.toString('utf8'))})`);
```
Comment (MikesGlitch) [inline packages/desktop-electron/index.ts:69]: Spitting the logs back out to the main browser window

**Comment by MikesGlitch** [comment]
image

**Review by MatissJanis — APPROVED** [review]
Nice! This will make debugging much easier

**Pull request merged**